apps/vanilla-three/src/main.js

```diff
@@ -0,0 +1,27 @@
+import * as THREE from '@jscadui/bundle-three'
+import { RenderThree } from '@jscadui/render-three'
+import { makeAxes, makeGrid } from '@jscadui/scene'
+import { light } from '@jscadui/themes'
+
+window.THREE = THREE // just so we can play with and inspect THREE in browser console
+let theme = light
```

Comment:
Remove theme, use light, or remove theme and use `{ light as theme }` above.

Destrcture the props: `const { grid1, bg, grid2 } = light`

Reply:
k